app/controllers/todos_controller.rb

```diff
@@ -78,7 +79,7 @@ def set_todo
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def todo_params
-      params.require(:todo).permit(:description, :owner, :user_id, :avatar, :nombre)
+      params.require(:todo).permit(:description, :owner, :user_id, :avatar, :nombre, :estate)
```

Comment:
Do you mean state or status? _estate_ means another thing

Reply:
estate is how i call mi variable :P

Reply:
Yeah, but the variable should be also descriptive, estate is nothing related to this context, rename it please to `status` or something that can tell us what is this variable used for.